lib/massager_commander.ex

```diff
@@ -0,0 +1,56 @@
+defmodule MassagerCommander do
+  @file_path_root "/ago/aa-work-solve/2022-09-05_SOLV-4646_minnowswim-shopify-historical-import"
+  @single_orders_file "/ago/aa-work-solve/2022-09-05_SOLV-4646_minnowswim-shopify-historical-import/import_massaging/orders_export_06.csv"
+  @all_orders_files [
+    "orders_export_01.csv",
+    "orders_export_02.csv",
+    "orders_export_03.csv",
+    "orders_export_04.csv",
+    "orders_export_05.csv",
+    "orders_export_06.csv",
+    "orders_export_07.csv",
+    "orders_export_08.csv",
+    "orders_export_09.csv",
+    "orders_export_10.csv",
+    "orders_export_11.csv",
+    "orders_export_12.csv"
+  ]
+
+  # ["returns_export_1.csv"]
+  @all_returns_files []
+
+  # ["transactions_export_1.csv"]
+  @all_transactions_files []
+
+  # MassagerCommander.massage_single()
+  def massage_single() do
```

Comment:
We've got a few functions here to do different things. It would be better if there was just one and we varied the parameters.